app/models/ab_test.rb

```diff
@@ -39,6 +39,35 @@ def self.load_assignments(metrics_uuid)
     where(enabled: true).map { |ab_test| ab_test.assignment(metrics_uuid) }.reject(&:flag_override?).map(&:tag)
   end
 
+  # determine if assignment for metrics_uuid qualifies for showing an updated feature
+  def override_feature?(metrics_uuid, groups: [])
+    return false unless enabled
+
+    groups.include? assignment(metrics_uuid).group_name
+  end
+
+  # determine what state a potential feature_flag and metrics_uuid combine to show
+  # will consider flag default state, possible user overrides, and A/B test state
```

Comment:
I had trouble understanding this docstring, as I initially read it all as a single sentence.  For clarity:

```suggestion
  # determine what state a potential feature_flag and metrics_uuid combine to show.
  # will consider flag default state, possible user overrides, and A/B test state
```

Prose-conventional uppercase ("Determine", "Will") would also help distinguish what's a new sentence vs. what's merely a line break, though maybe not totally necessary.

Reply:
Ah great point - I can do that moving forward.